SPECS/python-idna/python-idna.spec

```diff
@@ -1,6 +1,6 @@
 Summary:        Internationalized Domain Names in Applications (IDNA).
 Name:           python-idna
-Version:        3.3
+Version:        3.7
```

Comment:
As per:
https://github.com/advisories/GHSA-jjg7-2v4v-x38h

This seems to be a moderate, why are we fast-tracking it ?
[double checking .. maybe i might have missed some prior context]

Reply:
@mfrw NIST considers it a HIGH https://nvd.nist.gov/vuln/detail/CVE-2024-3651 that is why I think.

Reply:
Also.. we are more ok with fastt-tracking the moderates these days.  But not at the risk of a high or critical